doc_source/mesh-k8s-integration.md

```diff
@@ -1,10 +1,10 @@
 # Tutorial: Configure App Mesh integration with Kubernetes<a name="mesh-k8s-integration"></a>
 
-When you integrate AWS App Mesh with Kubernetes, you manage App Mesh resources, such as virtual services and virtual nodes, through Kubernetes\. You also automatically add the App Mesh sidecar container images to Kubernetes pod specifications\. This tutorial guides you through the installation of the following open source components that enable this integration:
-+ **App Mesh controller for Kubernetes** – The controller is accompanied by the deployment of three Kubernetes custom resource definitions: `mesh`, `virtual service`, and `virtual node`\. The controller watches for creation, modification, and deletion of the custom resources and makes changes to the corresponding App Mesh `[mesh](https://docs.aws.amazon.com/app-mesh/latest/userguide/meshes.html)`, `[virtual service](https://docs.aws.amazon.com/app-mesh/latest/userguide/virtual_services.html)` \(including `[virtual router](https://docs.aws.amazon.com/app-mesh/latest/userguide/virtual_routers.html)` and `[route](https://docs.aws.amazon.com/app-mesh/latest/userguide/routes.html)`\), and `[virtual node](https://docs.aws.amazon.com/app-mesh/latest/userguide/irtual_nodes.html)` resources through the App Mesh API\. To learn more or contribute to the controller, see the [GitHub project](https://github.com/aws/aws-app-mesh-controller-for-k8s)\.
-+  **App Mesh sidecar injector for Kubernetes** – The injector installs as a webhook and injects the following containers into Kubernetes pods that are running in specific, labeled namespaces\. To learn more or contribute, see the [GitHub project](https://github.com/aws/aws-app-mesh-inject)\.
-  + **App Mesh Envoy proxy** –Envoy uses the configuration defined in the App Mesh control plane to determine where to send your application traffic\.\. 
-  + **App Mesh proxy route manager **– The route manager sets up a pod’s network namespace with `iptables` rules that route ingress and egress traffic through Envoy\.
+When you integrate AWS App Mesh with Kubernetes, you manage App Mesh resources, such as meshs, virtual services, virtual nodes and virtual routers, through Kubernetes\. You also automatically add the App Mesh sidecar container images to Kubernetes pod specifications\. This tutorial guides you through the installation of the following open source components that enable this integration:
```

Comment:
Minor typo .. "meshs" -> "meshes"

Reply:
Already fixed

Reply:
General Comment: I think it is also better to add that deleting a Mesh will not automatically trigger delete of VirtualNodes/VirtualServices/VirtualRouters under that mesh. Other AppMesh K8S resources should be deleted prior to attempting a mesh delete. This is different to how the old controller behaves, so we should probably mention it.

Reply:
Good point on the mesh delete. I'll add it